### DISCUSSION TITLE
How to uninstall

### DISCUSSION THREAD

**Discussion by garethsnaim** [Rem0o/FanControl.Releases#749] [General]
Can someone tell me what I need to do to clean uninstall please, simply deleting the folder is not it sadly?

**Comment by Rem0o** [comment]
It's a portable app, there is nothing to uninstall.  You can just delete the folder and that's it. Make sure the app is closed before deleting.

**Comment by garethsnaim** [comment]
Hi thanks. When I ran for the first ever time the delay for start up was 30 seconds. I set it to 60

I deleted that folder and download again when run, that delay still says 60? That tells me there is a pref file somewhere?

**Reply by Rem0o** [reply]
Task Scheduler, there is a task named "FanControl". The checkbox and delay text input showcase the state of that task.

**Reply by garethsnaim** [reply]
Ok I will take a look.
